applications/common/backend/src/test/java/pl/ds/kyanite/common/components/models/table/TableCellComponentTest.java

```diff
@@ -47,16 +47,16 @@ void defaultTableCellComponentModelTest() {
         .adaptTo(TableCellComponent.class);
     assertThat(model).isNotNull();
     assertThat(model.getText()).isEqualTo("Content");
-    assertThat(model.getRows()).isEqualTo(1);
-    assertThat(model.getColumns()).isEqualTo(1);
+    assertThat(model.getRowspan()).isEqualTo(1);
+    assertThat(model.getColspan()).isEqualTo(12);
```

Comment:
Result of computation is visible also in this test.
The table has one column.
So now it received colspan=12